plugins/org.jkiss.dbeaver.ext.cubrid/src/org/jkiss/dbeaver/ext/cubrid/edit/CubridViewManager.java

```diff
@@ -0,0 +1,148 @@
+import org.jkiss.dbeaver.model.impl.edit.SQLDatabasePersistAction;
+import org.jkiss.dbeaver.model.impl.sql.edit.struct.SQLTableManager;
+import org.jkiss.dbeaver.model.runtime.DBRProgressMonitor;
+import org.jkiss.dbeaver.model.sql.SQLUtils;
+import org.jkiss.utils.CommonUtils;
+
+public class CubridViewManager extends GenericViewManager implements DBEObjectRenamer<GenericTableBase> {
+
+    @NotNull
+    @Override
+    protected GenericTableBase createDatabaseObject(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBECommandContext context,
+            @Nullable Object container,
+            @Nullable Object copyFrom,
+            @Nullable Map<String, Object> options) {
+        GenericStructContainer structContainer = (GenericStructContainer) container;
+        String tableName = getNewChildName(monitor, structContainer, SQLTableManager.BASE_VIEW_NAME);
+        GenericTableBase viewImpl = structContainer.getDataSource().getMetaModel().createTableOrViewImpl(
+                structContainer, tableName, GenericConstants.TABLE_TYPE_VIEW, null);
+        if (viewImpl instanceof GenericView) {
+            ((GenericView) viewImpl).setObjectDefinitionText("\n");
+        }
+        return viewImpl;
+    }
+
+    @Override
+    protected void addObjectCreateActions(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBCExecutionContext executionContext,
+            @NotNull List<DBEPersistAction> actions,
+            @NotNull ObjectCreateCommand command,
+            @NotNull Map<String, Object> options) {
+        createOrReplaceViewQuery(actions, command);
+    }
+
+    @Override
+    protected void addObjectModifyActions(
+            @NotNull DBRProgressMonitor monitor,
+            @NotNull DBCExecutionContext executionContext,
+            @NotNull List<DBEPersistAction> actionList,
+            @NotNull ObjectChangeCommand command,
+            @NotNull Map<String, Object> options) {
+        createOrReplaceViewQuery(actionList, command);
+    }
+
+    private void createOrReplaceViewQuery(
+            @NotNull List<DBEPersistAction> actions,
+            @NotNull DBECommandComposite<GenericTableBase, PropertyHandler> command) {
+        CubridView view = (CubridView) command.getObject();
+        StringBuilder query = new StringBuilder(200);
+        String viewDDL = view.getDDL();
+        boolean hasComment = command.hasProperty(DBConstants.PROP_ID_DESCRIPTION);
+        if (viewDDL == null) {
+            viewDDL = "";
+        }
+        if (!view.isPersisted()) {
+            query.append("CREATE VIEW " + view.getUniqueName() + "\nAS ");
+            query.append(viewDDL);
+            if (hasComment && view.getDescription() != null) {
+                query.append("\nCOMMENT = " + SQLUtils.quoteString(view, CommonUtils.notEmpty(view.getDescription())));
+            }
+        } else {
+            if (command.hasProperty("objectDefinitionText")) {

```

Comment:
Use `DBConstants.PARAM_OBJECT_DEFINITION_TEXT`

Reply:
I have fixed this.